runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -368,6 +368,9 @@ struct AMDAIEDeviceModel {
     return deviceConfig.maxVectorSizeBits;
   }
 
+  uint32_t getNumCoreRows() const { return rows() - 2; }
```

Comment:
https://github.com/nod-ai/iree-amd-aie/blob/cbb17c9397f59de1ebd9bfea67281f87fd8899d7/compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.cpp#L113

I'm wondering if there will might ever be more than 1 row of L2 memory tiles, or some other reason this might not work in the future

Reply:
It should just be `configPtr.AieTileNumRows`